core/src/impl/Kokkos_Profiling.hpp

```diff
@@ -45,12 +45,10 @@
 #ifndef KOKKOS_IMPL_KOKKOS_PROFILING_HPP
 #define KOKKOS_IMPL_KOKKOS_PROFILING_HPP
 
-#include <Kokkos_Core_fwd.hpp>
-#include <Kokkos_ExecPolicy.hpp>
-#include <Kokkos_Macros.hpp>
-#include <Kokkos_Tuners.hpp>
 #include <impl/Kokkos_Profiling_Interface.hpp>
+#include <impl/Kokkos_Command_Line_Parsing.hpp>
```

Comment:
```suggestion
#include <impl/Kokkos_Command_Line_Parsing.hpp>
```

Reply:
I'm sorry, I can't figure out what this diff is asking for?

Reply:
Not include it here.  Unless I am missing something and you need it in that very header.

Reply:
132509d